testing/unit-testing-jest/docs/issues.md

```diff
@@ -0,0 +1,141 @@
+## Issues
+
+### i18n dependency
+
+#### Issue:
+If you test suit fails to run, with the following error message:
+```ts
+● Test suite failed to run
+
+    It looks like your application or one of its dependencies is using i18n.
+    Angular 9 introduced a global `$localize()` function that needs to be loaded.
+    Please run `ng add @angular/localize` from the Angular CLI.
+    (For non-CLI projects, add `import '@angular/localize/init';` to your `polyfills.ts` file.
+    For server-side rendering applications add the import to your `main.server.ts` file.)
+```
+
+#### Solution:
+This mean that your module uses i18n, you'll need to expose the ```$localize``` function in the global namespace. This need to be imported on the ```test-setup.ts``` file.
+
+```ts
+import 'jest-preset-angular';
+import '@angular/localize/init';
+```
+
+### Navigator dependency
+Sometimes, when we are testing many problems don’t let our test execute. Is so frustrating don't find a simple solution just checking the error console and understanding the problem. This is a collection of typical errors that you can get when you are testing. Hope that helps.
+
+### Error with the navigator
+#### Issue:
+If we use the navigator router on the test files, maybe we get an error like this:
+![error navigator](./assets/NavigatorError.png "Error");
+This means we are using a specific path to test the routing. But we aren’t declaring on the environments file.
```

Comment:
```suggestion
This means we are using a specific path to test the routing. But we aren't declaring on the environments file.
```
Avoid to use this char because markdown things is a scape char xD